vlm/src/main/scala/geotrellis/contrib/vlm/spark/RasterSourceRDD.scala

```diff
@@ -41,7 +45,15 @@ object RasterSourceRDD {
     readingSources: Seq[ReadingSource],
     layout: LayoutDefinition,
     partitionBytes: Long
-  )(implicit sc: SparkContext): MultibandTileLayerRDD[SpatialKey] = {
+  )(implicit sc: SparkContext): MultibandTileLayerRDD[SpatialKey] =
+    readPartitionBytesK[SpatialKey](readingSources, layout, (_, sk) => sk, partitionBytes)
```

Comment:
I decided to add `K` overloads in order to make this change smoother, at least in the beginning.